### DISCUSSION TITLE
Expectation for comparing counts on two tables or two SQL results

### DISCUSSION THREAD

**Discussion by raangs** [great-expectations/great_expectations#5688] [Expectations]
We have ETL pipelines, and we want to implement some validations on daily ingested ETL data with source tables.

Is there any Expectation i can use, to compare the counts between two tables (preferably using two SQLs) like source table count with target table count.

**Comment by austiezr** [comment]
Hey @raangs ! Thanks for reaching out. Take a look at [this documentation on making cross-table comparisons](https://docs.greatexpectations.io/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_user_configurable_profiler).

Additionally, take a look at these expectations ([1](https://greatexpectations.io/expectations/expect_table_row_count_to_equal_other_table), [2](https://greatexpectations.io/expectations/expect_table_checksum_to_equal_other_table)) that may be relevant.
